frontend/src/App.js

```diff
@@ -1,38 +1,56 @@
-import Home from './page/Home'
-import Explore from './page/Explore'
-import Contribute from './page/Contribute'
-import Login from './page/Login'
-import Signup from './page/Signup'
-import {Route, Routes,} from 'react-router-dom'
-import Navbar from './component/Navbar'
-import Footer from './component/Footer'
+import Home from "./page/Home";
+import React, { useEffect, createContext, useReducer, useContext } from "react";
+import Explore from "./page/Explore";
+import Contribute from "./page/Contribute";
+import Login from "./page/Login";
+import Signup from "./page/Signup";
+import { Route, Routes, useNavigate } from "react-router-dom";
+import Navbar from "./component/Navbar";
+import Footer from "./component/Footer";
+import Song from "./page/Song";
+import { reducer, initialState } from "./reducers/userReducer";
+
+export const UserContext = createContext();
+
+const Routing = () => {
+  const Navigate = useNavigate();
```

Comment:
general feedback on naming convention -> Variables, FileNames and Function should be in `camelCase` and Components and Component File should be in `PascalCase`.